File: DFRobot_RGBLCD/python/Cursor/Cursor.py
import sys
sys.path.append('../')
import time
import rgb1602

lcd = rgb1602.RGB1602(16,2)

lcd.print("hello, world!")

while True:
  # Turn off the cursor:
  lcd.noCursor();
  time.sleep(0.5);
  # Turn on the cursor:
  lcd.cursor();
  time.sleep(0.5);

File: cpp/examples/Cursor.cpp
#include "../DFRobot_RGBLCD.h"

int main(){
  DFRobot_RGBLCD lcd(16, 2);
  lcd.init();

  lcd.print("Hello World!");
  
  while(1){
    // Turn off the cursor:
    lcd.noCursor();
    delay(500);
    // Turn on the cursor:
    lcd.cursor();
    delay(500);
  }
  return 0;
}
